contracts/CrossChain.sol

```diff
@@ -349,6 +349,14 @@ contract CrossChain is Config, Initializable, ICrossChain {
         emit Reopened(msg.sender);
     }
 
+    function emergencyChangeSequence(bool isSendSequence, uint8 channelId, bool isIncrease) external onlyEmergencyUpgradeOperator {
```

Comment:
Please change to `emergencyChangeSequence(uint8 channelId, bool isSendSequence, bool isIncrease)`

Reply:
fixed